actions/helixUtils.js

```diff
@@ -35,6 +35,20 @@ class HelixUtils {
         return { PREVIEW, LIVE };
     }
 
+    getRepo(isFloodgate = false, fgColor = 'pink') {
+        return isFloodgate ? `${urlInfo.getRepo()}-${fgColor}` : urlInfo.getRepo();
+    }
+
+    getAdminApiKeys(isFloodgate = false, fgColor = 'pink') {
```

Comment:
As this is returning a boolean, could the function be renamed? `isAdminApiKeyAvailable()` or something similar?

Reply:
Thanks @sukamat , Have updated the function name as suggested.